Docs: Added unofficial Windows install instructions (#786)
files-community/Website: src/routes/docs/install/+page.md

### PowerShell

> 1. Open the default PowerShell (Powershell 7 will not work) as administrator

Reviewer: ```suggestion
1. Open the default PowerShell (Powershell 7 will not work) as administrator.
```

> 1. Open the default version of PowerShell as administrator (Powershell 7 will **not** work).

Reviewer 2: The cross-platform version of PowerShell is called PowerShell Core and the word "Core" at the end is used to differentiate it from its .NET Framework counterpart and predecessor, named Windows PowerShell.

> Originally only for Windows, known as **Windows PowerShell**, it was made [open-source](https://en.wikipedia.org/wiki/Open-source_software) and [cross-platform](https://en.wikipedia.org/wiki/Cross-platform) on August 18, 2016, with the introduction of **PowerShell Core**.[[9]](https://en.wikipedia.org/wiki/PowerShell#cite_note-ars-opensource-9) The former is built on the [.NET Framework](https://en.wikipedia.org/wiki/.NET_Framework); the latter on [.NET](https://en.wikipedia.org/wiki/.NET) (previously .NET Core).

(PowerShell Wikipedia article)

> 1. Open the default version of PowerShell as administrator (Powershell 7 will **not** work).

Author: It think making combing it to `Open Windows PowerShell as administrator (PowerShell 7 will **not** work).` would be best as those are now the actual names used now

2. For Stable use this command
   ```
   Add-AppxPackage -AppInstallerFile https://cdn.files.community/files/stable/Files.Package.appinstaller
   ```

>    For Preview use this command

Reviewer: ```suggestion
   For the preview branch, use the following command
```

   ```
   Add-AppxPackage -AppInstallerFile https://cdn.files.community/files/preview/Files.Package.appinstaller
   ```

> 3. Wait until the blue bar at the top doesn't show

Reviewer: ```suggestion
3. Wait until the blue bar at the top disappears 
```

scoop install nonportable/files-np
```

### Offline installers

<InfoBar title="Note" severity="information">
	The files will need to be installed in the same order as below
</InfoBar>

```

> https://cdn.files.community/files/stable/Files.Package_4.0.0.0_Test/Dependencies/x64/Microsoft.VCLibs.x64.14.00.Desktop.appx

Reviewer: This is the `x64` version, should we also include links for `arm64`?

Author: I added the arm links, don't know the formatting to make them collapsible

Author: Can you confirm it looks good to you

### Windows cannot install package because a different package with the same name is already installed

If you get this error code, you'll need to uninstall and [reinstall](/download/) Files. You can also export/import the settings file to keep your existing settings.

> ### Unoffical Windows installs
> This includes but is not limited to AtlasOS, ReviOS, disabling Windows update, removing the Microsoft store. These are not supported but you can try the below. If they don't work and you have used one of these you won't be able to use Files without reinstalling Windows using the official installer from https://www.microsoft.com/software-download/windows11. 

Reviewer: ```suggestion
### Unofficial and modified versions of Windows

This includes but is not limited to AtlasOS, ReviOS, disabling Windows update, removing the Microsoft store. Files is not officially supported on devices with these modifications.
```

> 1. Type Services in Windows start menu

Reviewer: Should this be under its own section?

Author: Where else could it go? Since the instructions would only apply if Windows was modify in an unsupported way

Reviewer: It can stay here, but we can add an introduction on when users should take these steps.

Reviewer: > I can get the error message if you wanted

That would be helpful, ideally, we would list the error as another section and put these steps underneath it.

Author: Theres a few errors I have seen this being the issue

- Class not registered
- The text associated with this error could not be found
- A service wasn't able to be started

Author: How does this sound

### Unofficial Windows installs
This includes but is not limited to using AtlasOS, ReviOS, disabling Windows update, uninstalled the Microsoft Store, seeing errors when installing like, Class not registered, The text associated with this error could not be found or A Prerequisite for install could not be satisfied. These are not supported but you can try the below

Reviewer: I think this needs a little more context.
"Unofficial or modified versions of Windows are not support. This includes removing the Microsoft Store, disabling Windows updates etc."
